ddtrace/internal/telemetry/writer.py

```diff
@@ -502,6 +500,51 @@ def add_log(self, level, message, stack_trace="", tags=None):
             # Logs are hashed using the message, level, tags, and stack_trace. This should prevent duplicatation.
             self._logs.add(data)
 
+    def add_integration_error_log(self, msg, exc, warning=False):
+        if warning is False:
+            log.debug(msg, exc_info=exc)
+        else:
+            log.warning(msg, exc_info=exc)
```

Comment:
It would be nice if we could collect telemetry logs without impacting users. Some log messages may not be useful to users.

```suggestion
       if not internal:
          if warning is False:
              log.debug(msg, exc_info=exc)
          else:
              log.warning(msg, exc_info=exc)
```

Reply:
I let the `log.debug/log.warning` because they were there before but I'm good with completely deleting them. What do you think ? If the goal of `internal` is to allow a dev to see the log we can also remove the call and let the dev adds them when needed during dev.

Reply:
For awareness: https://dd.slack.com/archives/C075UAQ0L1Y/p1743768739419159